src/graphdb.js

```diff
@@ -61,33 +80,48 @@ function graphDBInferenceSchema (graphbSchema, addMutations) {
 
     gdbs.nodeStructures.forEach(node => {
         // node type
-        let nodeCase = node.label;
+        let nodeCase = replaceCleanseLabel(node.label);
         if (changeCase) {
-            nodeCase = toPascalCase(node.label); 
+            nodeCase = toPascalCase(nodeCase);
+        }
+
+        if (node.label !== nodeCase) {
             r += `type ${nodeCase} @alias(property:"${node.label}") {\n`;
-        } else {
+        }
+        else {
             r += `type ${nodeCase} {\n`;
         }
         
         r += '\t_id: ID! @id\n';
 
         node.properties.forEach(property => {
-            if (property.name == 'id')
+            if (property.name == 'id') {
                 r+= `\tid: ID\n`;
-            else
-                r+= `\t${property.name}: ${property.type}\n`;
+            }
+            else {
+                let propertyCase = replaceCleanseLabel(property.name);
+                let alias = '';
+                if (property.name !== propertyCase) {
+                    alias = ` @alias(property: "${property.name}")`;
+                }
+                r+= `\t${propertyCase}: ${property.type}${alias}\n`;
+            }
         });
         
         let edgeTypes = [];
         gdbs.edgeStructures.forEach(edge => {            
             edge.directions.forEach(direction => {
-                let fromCase = toPascalCase(direction.from);
-                let toCase = toPascalCase(direction.to);
-                let edgeCase = toPascalCase(edge.label);
+                let fromCase = replaceCleanseLabel(direction.from);
+                fromCase = toPascalCase(fromCase);
+                let toCase = replaceCleanseLabel(direction.to);
+                toCase = toPascalCase(toCase);
+                let edgeCase = replaceCleanseLabel(edge.label);
+                edgeCase = toPascalCase(edgeCase);
```

Comment:
Thanks for the tip! For these variables, they need to use `let` since it's in a loop that loops through every direction and edge so it replaces the variable with them. I will rewrite it so `replaceCleanseLabel` is called inside `toPascalCase` to make it look nicer.